examples/README.md

```diff
@@ -1,12 +1,13 @@
 # Simple examples
 
-Following examples are show casing using of live updates via Server-sent events (SSE) and Websocket (WS). Shows how to use public as well as private delivery.
+Following examples are show casing using of live updates via Server-sent events (SSE), Longpolling and Websocket (WS). Shows how to use public as well as private delivery.
 
-## SSE
+# SSE
+## Example 1: SSE
 
 This example shows simplest scenario when all messages will arrive to UI.
 
-> examples/sse-demo.html
+> examples/sse-1-demo.html
```

Comment:
What is the intention here with the changes?

Reply:
I was a bit struggling to find the right file to the examples so I numbered them according the the Example-no. :) what do you think?

Reply:
Ok, I guess that makes sense.